tools/include/eosio/codegen.hpp

```diff
@@ -311,7 +313,7 @@ namespace eosio { namespace cdt {
             static std::set<std::string> _notify_set; //used for validations
             if (decl->isEosioAction()) {
                name = generation_utils::get_action_name(decl);
-               validate_name(name, [&]() {emitError(*ci, decl->getLocation(), "action not a valid eosio name");});
+//               validate_name(name, [&]() {emitError(*ci, decl->getLocation(), "action not a valid eosio name");});
```

Comment:
It doesn't seem to take out validation of names for everyone even if they are not using the new hashed name system.